enhancements/ptp-ha.md

```diff
@@ -0,0 +1,82 @@
+---
+title: ptp_multi-nic_ha_support
+authors:
+  - "@danielmellado"
+reviewers:
+  - "@zshi-redhat"
+  - "@SchSeba"
+  - "@aneeshkp"
+  - "@imiller0"
+approvers:
+  - "@ijolliffe"
+creation-date: 2022-06-15
+last-updated: 2022-06-15
+tracking-link: TBD
+status: provisional
+---
+
+# PTP multi-nic HA Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-nic support for PTP operator [1] seems to be already built-in. In
+enterprise telco environments, there's often a need to load balance two
+different signals from different PTP IEEE 1588-2008 sources.
+
```

Comment:
As a first glance, after my chat with Ken and his team I thought about going for a 'simplest-ever' first kind of approach. Basically if one of the PTP daemons doesn't reply, switch to the other one. I'll update this to cover that. Were you thinking about other specific HA approaches?

Reply:
the nic's in question are multi interface - the implications and provisioning required of the ports on the HA provided.  it would be good to clarify the requirement relative to card level HA and make clear what is being covered.

Reply:
If you see on line 95 I also ask the same question. Let's address this in the meetings I proposed